src/enclave_proc/commands.rs

```diff
@@ -71,6 +71,22 @@ pub fn run_enclaves(
         e.add_subaction("Failed to construct EnclaveManager with given arguments".to_string())
     })?;
 
+    let mut signature_checker = PcrSignatureChecker::from_eif(&args.eif_path).map_err(|e| {
+        new_nitro_cli_failure!(
+            &format!("Failed to create signature checker: {:?}", e),
```

Comment:
Could use "create EIF signature checker".

Reply:
Followed all suggestions and modified.